spec/03-plugins/25-response-rate-limiting/04-access_spec.lua

```diff
@@ -92,7 +92,7 @@ end
 for _, strategy in helpers.each_strategy() do
 for _, policy in ipairs({"local", "cluster", "redis"}) do
 
-describe(fmt("#flaky Plugin: response-ratelimiting (access) with policy: #%s [#%s]", policy, strategy), function()
+describe(fmt("flaky Plugin: response-ratelimiting (access) with policy: #%s [#%s]", policy, strategy), function()
```

Comment:
Since you verified that tests pass, these can be reverted so that it's `#flaky` again